src/App.jsx

```diff
@@ -13,34 +14,89 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPreparedGoods(goods, sortParameter) {
+  if (sortParameter) {
+    goods.sort((a, b) => {
+      switch (sortParameter) {
+        case SORT_BY_ALPHABET:
+          return a.localeCompare(b);
+        case SORT_BY_LENGTH:
+          return a.length - b.length;
+        default:
+          return 0;
+      }
+    });
```

Comment:
The `goods.sort()` method mutates the original array. Consider creating a copy of the array before sorting to avoid side effects. This can be done by using the spread operator or `Array.from()`.